<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduWidgetTaxon</class>
 <widget class="QGroupBox" name="AbulEduWidgetTaxon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>9.2 Chemin taxon</string>
  </property>
  <layout class="QVBoxLayout" name="vlThis">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSource">
       <property name="text">
        <string>9.2.1 Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSource"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbLOM9_2_2">
     <property name="title">
      <string>9.2.2 Taxon</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lblId1">
        <property name="text">
         <string>9.2.2.1 ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leId1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblEntry1">
        <property name="text">
         <string>9.2.2.2 Entr√©e</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leEntry1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPlusLOM9_2_2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
